eng/Directory.Packages.props

```diff
@@ -46,10 +46,10 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(DotNetBuildSourceOnly)' != 'true' and '$(TargetFramework)' == 'net472'">
-    <PackageVersion Include="Microsoft.Build" Version="17.13.9" />
-    <PackageVersion Include="Microsoft.Build.Framework" Version="17.13.9" />
-    <PackageVersion Include="Microsoft.Build.Tasks.Core" Version="17.13.9" />
-    <PackageVersion Include="Microsoft.Build.Utilities.Core" Version="17.13.9" />
+    <PackageVersion Include="Microsoft.Build" Version="17.15.0-preview-25279-04" />
+    <PackageVersion Include="Microsoft.Build.Framework" Version="17.15.0-preview-25279-04" />
+    <PackageVersion Include="Microsoft.Build.Tasks.Core" Version="17.15.0-preview-25279-04" />
+    <PackageVersion Include="Microsoft.Build.Utilities.Core" Version="17.15.0-preview-25279-04" />
```

Comment:
Did you actually need to bump these?

Reply:
It was literally what the tool told me to do :)